code/modules/mob/living/carbon/alien/special/facehugger.dm

```diff
@@ -27,10 +28,72 @@ var/const/MAX_ACTIVE_TIME = 400
 	var/strength = 5
 
 	var/attached = 0
+	var/TARGETTIME = 0.5 // seconds
```

Comment:
Weird to have these variables all uppercase, since we generally use uppercase for globals / defines.